lib/rucio/daemons/reaper/reaper2.py

```diff
@@ -145,11 +146,13 @@ def get_rses_to_process(rses, include_rses, exclude_rses, vos):
     return rses
 
 
-def delete_from_storage(replicas, prot, rse_info, staging_areas, auto_exclude_threshold, logger=logging.log):
+def delete_from_storage(replicas, prot, rse_info, staging_areas, auto_exclude_threshold, bulk, logger=logging.log):
```

Comment:
See comment above, if we assume everything with globus is bulk, we don't need that part.